src/herder/TxSetFrame.cpp

```diff
@@ -461,6 +461,32 @@ TxSetFrame::sizeOp() const
         });
 }
 
+int64_t
+TxSetFrame::getBaseFee(LedgerHeader const& lh) const
+{
+    int64_t baseFee = lh.baseFee;
+    if (lh.ledgerVersion >= 11)
+    {
+        size_t ops = 0;
+        int64_t lowBaseFee = std::numeric_limits<int64_t>::max();
+        for (auto& txPtr : mTransactions)
+        {
+            auto txOps = txPtr->getEnvelope().tx.operations.size();
+            ops += txOps;
+            int64_t txBaseFee =
+                bigDivide(txPtr->getFeeBid(), 1, txOps, Rounding::ROUND_UP);
```

Comment:
do we expect fee bids so big that bigDivide would be required to handle them?

Reply:
ah yes, we just need a round_up int64 divide here.